testutil/constants/constants.go

```diff
@@ -129,3 +97,48 @@ var (
 		},
 	}
 )
+
+// GetExampleChainCoinInfo returns appropriate EvmCoinInfo for testing based on chainID.
+// Deprecated: Use testutil/config.CreateEvmCoinInfoFromDynamicConfig() instead.
+// This function is kept for backward compatibility but should be replaced
+// with dynamic configuration generation.
```

Comment:
if we are deprecating this and keeping it for compatibility, then why not just keep the global map type?

Folks will be broken because we've changed the API

Reply:
would also make the diff much smaller